src/main/resources/anniv5Resources/localization/eng/dimensiongatepack3/Cardstrings.json

```
@@ -16,8 +16,8 @@
   },
   "${ModID}:Ouroboros": {
     "NAME": "Ouroboros",
-    "DESCRIPTION": "At the end of each turn, if you fully blocked any source of damage, gain 1 Strength.",
-    "UPGRADE_DESCRIPTION": "Innate. NL At the end of each turn, if you fully blocked any source of damage, gain 1 Strength."
+    "DESCRIPTION": "The first time you fully block any source of damage each turn, gain 1 Strength.",
+    "UPGRADE_DESCRIPTION": "Innate. NL The first time you fully block any source of damage each turn, gain 1 Strength."
```

Since this was changed from "blocked" to "block" it will also need the zero-width space treatment.
```suggestion
    "UPGRADE_DESCRIPTION": "Innate. NL The first time you fully block any source of damage each turn, gain 1 Strength."
```